tests/TestHelpers/OcisHelper.php

```diff
@@ -38,13 +40,20 @@ public static function isTestingOnOcis() {
 		return (\getenv("TEST_OCIS") === "true");
 	}
 
+	/**
+	 * @return bool
+	 */
+	public static function deleteUserDataOcis() {
```

Comment:
```suggestion
	public static function getDeleteUserDataCommand() {
```
it does not need to be for ocis only